LandLord-latest/src/main/java/biz/princeps/landlord/OwnedLand.java

```diff
@@ -67,7 +66,7 @@ public String getOwnersString() {
         for (UUID uuid : itt) {
             names.add(Bukkit.getOfflinePlayer(uuid).getName());
         }
-        return itToString(names.iterator());
+        return String.join(", ", names);
```

Comment:
add a `private static final String NAMES_DELIMITER = ", "` (or something similar) to avoid magic values and reuse it in the other places.
Edit: See the comment in the legacy class too